### PR TITLE
NEXTEUROPA-12244: Disable and uninstall xmlsitemap_i18n module.

### PR DESCRIPTION
**Pull request by voidtek**
https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-12244

### CODE REVIEW

**Comment by dxvargas** [comment]
👍

File: tests/features/sitemap.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Check the Sitemap
+  In order to check if the sitemap is available and correct.
+  As an administrator
+  I want to check the sitemap generated is available and correct.
```
Comment (cyberwolf) [inline tests/features/sitemap.feature]: These are not business goals. A sitemap is not there in order to be able to check if it's available and correct.

File: profiles/common/modules/features/sitemap/sitemap.install [hunk 2]
```diff
@@ -85,3 +78,11 @@ function sitemap_update_7120() {
 
   cache_clear_all('variables', 'cache_bootstrap');
 }
+
+/**
+ * NEXTEUROPA-12244: Disable and uninstall xmlsitemap_i18n module.
+ */
+function sitemap_update_7121() {
+  module_disable(array('xmlsitemap_i18n'), FALSE);
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/sitemap.install]: This does not offer any guarantuees that the xmlsitemap_i18n module won't be enabled again by a subsite builder on a new website.

File: profiles/common/modules/features/sitemap/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+The module aims to provide a Sitemap (XML).
+
+# Installation
+
+The feature can be activated using feature set menu #
+(/admin/structure/feature-set).
+
+# Usage
+
+When feature is enabled, the /sitemap.xml is available.
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/README.md]: Pay attention to the articles: ... the feature ...

File: profiles/common/modules/features/sitemap/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Sitemap Feature
+======================
+
+The module aims to provide a Sitemap (XML).
+
+# Installation
+
+The feature can be activated using feature set menu #
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/README.md]: Pay attention to the articles: ... the feature set menu ...

File: profiles/common/modules/features/sitemap/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+(/admin/structure/feature-set).
+
+# Usage
+
+When feature is enabled, the /sitemap.xml is available.
+
+The feature provides:
+- list of XML sitemap,
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/README.md]: ... a list of XML sitemaps ...

File: profiles/common/modules/features/sitemap/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- Search engines,
+- Administration page (Setting),
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/README.md]: ... an administration page

File: profiles/common/modules/features/sitemap/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+- Manually Rebuild links.
+
+## Regenerate the XML sitemap
+The feature regenerates the XML sitemap files using the cron or it's possible to
+regenerate manually.
+
+## Multilingual
+The feature includes the tags (rel = "alternate") for assuming the various
```
Comment (cyberwolf) [inline profiles/common/modules/features/sitemap/README.md]: "for assuming the various languages" is not clear to me what you mean with this, can this be clarified?

**Comment by cyberwolf** [comment]
👎

File: profiles/common/modules/features/sitemap/sitemap.install (hunk 2 above)
Comment (voidtek) [inline profiles/common/modules/features/sitemap/sitemap.install]: What is your proposition?

File: tests/features/sitemap.feature (hunk 1 above)
Comment (voidtek) [inline tests/features/sitemap.feature]: What is your proposition?

**Comment by dxvargas** [comment]
👍

**Comment by Fefaine** [comment]
:+1:

**Pull request merged**